util/src/main/i18n/templates/match.properties

```diff
@@ -176,6 +176,9 @@ broadcast.gameOver.teamWinner = {0} wins!
 # {0} = winning team name (e.g. plural names, "Attackers" or "Defenders")
 broadcast.gameOver.teamWinners = {0} win!
 
+# {0} = the amount of winners, always a number (e.g. "5 winners")
+broadcast.gameOver.winners = {0} winners!
```

Comment:
I'd rather it just say "Tied!".

Examples:

`winners.size() == 0`
```
Tied!
```

`winners.size() <= 3` (use [`TextComponent#list`](https://github.com/PGMDev/PGM/blob/96bed42423d7802e74c0b608e8ab1f496885058d/util/src/main/java/tc/oc/pgm/util/text/TextFormatter.java#L41))
```
Tied!
ElectroidFilms and Pablete1234
```

```
Tied!
ElectroidFilms, Pablete1234, and KingOfSquares
```

`winners.size() > 3` (refactor [`TextComponent#list`](https://github.com/PGMDev/PGM/blob/96bed42423d7802e74c0b608e8ab1f496885058d/util/src/main/java/tc/oc/pgm/util/text/TextFormatter.java#L41) to have a `maxSize` option, if over, adds the "and N more" to the end)
```
Tied!
ElectroidFilms, Pablete1234, and 3 more
```

Reply:
I believe the `and 3 more` to be a bad idea, you either show all the winners or show none, they all tied and so should have the same "recognition", there's no fair way to sort them better-to-worse (or else, you'd just have the highest-sorted as the one true winner)

Reply:
@Pablete1234 Fair enough, so we can leave out the "and more".

@KingOfSquares If it's more than 3 winners, don't show any subtitle.